Sources/VLCSettingsTableHeaderView.swift

```diff
@@ -0,0 +1,64 @@
+/*****************************************************************************
+ * VLCSettingsTableHeaderView.swift
+ * VLC for iOS
+ *****************************************************************************
+ * Copyright (c) 2018 VideoLAN. All rights reserved.
+ * $Id$
+ *
+ * Authors: Mike JS. Choi <mkchoi212 # icloud.com>
+ *
+ * Refer to the COPYING file of the official project for license.
+ *****************************************************************************/
+
+import UIKit
+
+class VLCSettingsTableHeaderView: UITableViewHeaderFooterView {
+    let headerHeight: CGFloat = 1.0
+    
+    @objc static let identifier = String(describing: VLCSettingsTableHeaderView.self)
+    @objc var headerLabel: UILabel!
+    @objc var separator: UIView!
+    
```

Comment:
What do you think about initializing them directly here?
For example: 

```

    @objc var separator: UIView = {
        let separator = UIView()
        separator.backgroundColor = PresentationTheme.current.colors.separatorColor
        separator.translatesAutoresizingMaskIntoConstraints = false
        return separator
    }()
```
This will make the init clearer by only having the `addSubView()` calls and the constraints stuff.

Reply:
Good point. I always forget you can do this :D